README.md

```diff
@@ -1,32 +1,64 @@
+Nothing further is necessary. Simply `using MKL` is enough to make Julia use Intel MKL:
+
+```julia
+julia> using LinearAlgebra
+
+julia> BLAS.get_config()
+LinearAlgebra.BLAS.LBTConfig
+Libraries: 
+└ [ILP64] libopenblas64_.so
+
+julia> using MKL
+
+julia> BLAS.get_config()
+LinearAlgebra.BLAS.LBTConfig
+Libraries: 
+└ [ILP64] libmkl_rt.so
+```
+
+Note that you can put `using MKL` into your `startup.jl` to make Julia automatically use Intel MKL in every session.
+
+### Using a preinstalled Intel MKL
+
+If you already have an Intel MKL installation available (as on most HPC clusters), you can use the the environment variable `JULIA_MKL_PATH` or the [preference](https://github.com/JuliaPackaging/Preferences.jl) `mkl_path` to hint MKL.jl to the `libmkl_rt` library. Specifically, the options are:
+
+* `mkl_jll` (default): Download and install MKL via [MKL_jll.jl](https://github.com/JuliaBinaryWrappers/MKL_jll.jl).
+* `system`: The package will try to automatically locate the libmkl_rt library (i.e. find it on the linker search path).
+* `path/to/my/libmkl_rt.<EXT>`: Explicit path to the `libmkl_rt.<EXT>` where `<EXT>` is the shared library extension of the system at hand (e.g. `.so`, `.dll`, `.dylib`)
+
+Note that, in contrast to the preference, the environment variable only has an effect when MKL.jl is (re-)precompiled. To force a change of the MKL path after the compilation has happened, use the function `MKL.set_mkl_path`, which takes the options listed above as input.
+
+
+## Installation (Julia 1.6 and older)
+
+On older releases of Julia, a new system image build happens right after installation. If it doesn't (happens when MKL_jll.jl has been installed before), run the following command to build a new system image and restart Julia.
 ```julia
 julia>] build MKL
 ```
 
-## To Check Installation:
+Afterwards, Julia is (always) using Intel MKL instead of OpenBLAS:
 
 ```julia
 julia> using LinearAlgebra
 
-julia> BLAS.vendor()  # Prior to Julia v1.7
+julia> BLAS.vendor() # replaced by BLAS.get_config() in Julia versions >= 1.7
 :mkl
-
-julia> BLAS.get_config()  # Julia v1.7 and later
-LinearAlgebra.BLAS.LBTConfig(LinearAlgebra.BLAS.LBTLibraryInfo[LinearAlgebra.BLAS.LBTLibraryInfo("/Users/viral/.julia/artifacts/073ff95e2c63501547247d6e1321bf4ee2a78933/lib/libmkl_rt.1.dylib", Ptr{Nothing} @0x00007fb6a5ef4820, "", UInt8[0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff  …  0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x01], :ilp64, :plain)], [:f2c_capable], ["LAPACKE_c_nancheck", "LAPACKE_cbbcsd", "LAPACKE_cbbcsd_work", "LAPACKE_cbdsqr", "LAPACKE_cbdsqr_work", "LAPACKE_cgb_nancheck", "LAPACKE_cgb_trans", "LAPACKE_cgbbrd", "LAPACKE_cgbbrd_work", "LAPACKE_cgbcon"  …  "zunmlq_", "zunmql_", "zunmqr_", "zunmr2_", "zunmr3_", "zunmrq_", "zunmrz_", "zunmtr_", "zupgtr_", "zupmtr_"])
 ```
 
-## Using the 64-bit vs 32-bit version of MKL
+To revert back to OpenBLAS you currently need to reinstall Julia.
```

Comment:
This is only true for Julia 1.6-, for Julia 1.7+, you just need to run `LinearAlgebra.BLAS.lbt_forward(OpenBLAS_jll.libopenblas_path, 1)`

Reply:
Well, that's why it is positioned under the "1.6 and older" section 😄. But I made it more explicit in a newer commit. Also, for 1.7+, I now mention

```julia
using OpenBLAS_jll
LinearAlgebra.BLAS.lbt_forward(OpenBLAS_jll.libopenblas_path; clear=true);
```

as a way to revert back. After this PR is merged I will try to properly address #58 in a new PR.